src/jdk.incubator.jextract/share/classes/jdk/internal/clang/Cursor.java

```diff
@@ -335,26 +335,15 @@ public Cursor getSpecializedCursorTemplate() {
         return new Cursor(Index_h.clang_getSpecializedCursorTemplate(cursor));
     }
 
```

Comment:
This diff removes the callback cache fix for jextract performance. I believe this is unintentional merge artefact.